configs/projects/_shared-agent-components.rb

```diff
@@ -38,8 +38,8 @@
 proj.component 'libxml2' unless platform.is_windows?
 proj.component 'libxslt' unless platform.is_windows?
 
-proj.component 'ruby-augeas' unless platform.is_windows?
-proj.component 'ruby-shadow' unless platform.is_aix? || platform.is_windows?
+proj.component 'ruby-augeas' unless platform.is_windows? || platform.name == 'solaris-113-sparc'
+proj.component 'ruby-shadow' unless platform.is_aix? || platform.is_windows? || platform.name == 'solaris-113-sparc'
```

Comment:
Why are these two components left out?

Reply:
Ah yeah just wanted to get the base runtime working and then come back to these. Same of C++ components below. I would ticket those out... :grin: